androidApp/src/main/java/com/mbta/tid/mbta_app/android/location/ViewportProvider.kt

```diff
@@ -26,20 +27,18 @@ import com.mbta.tid.mbta_app.android.util.isRoughlyEqualTo
 import com.mbta.tid.mbta_app.map.MapDefaults
 import com.mbta.tid.mbta_app.model.Stop
 import com.mbta.tid.mbta_app.model.Vehicle
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.withContext
 
-class ViewportProvider
-@OptIn(MapboxExperimental::class)
-constructor(var viewport: MapViewportState, isManuallyCentering: Boolean = false) {
+class ViewportProvider(var viewport: MapViewportState, isManuallyCentering: Boolean = false) {
```

Comment:
The APIs we're using here have been stabilized, so since I was here anyway I decided to go ahead and remove the `@OptIn`s. I haven't cascaded that outwards, though.